astronomy/orbital_elements.hpp

```diff
@@ -138,6 +147,10 @@ class OrbitalElements {
       MassiveBody const& primary,
       Body const& secondary);
 
+  template<typename PrimaryCentred>
+  static std::vector<Length> Distances(
+      DiscreteTrajectory<PrimaryCentred> const& trajectory);
```

Comment:
`Distances` is about as uninformative as it gets; make it `DistancesFromPrimary`.

Reply:
`RadialDistances`, and clarified `periapsis_distance` (as opposed to altitude).